First Chapter Code Sample Doesn't Compile

The example on the first chapter fails to compile:
```
use ggez;
use ggez::{conf, event, Context, GameResult};

// This struct will hold all our game state
// For now there is nothing to be held, but we'll add
// things shortly.
struct Game {}

// This is the main event loop. ggez tells us to implement
// two things:
// - updating
// - rendering
impl event::EventHandler for Game {
    fn update(&mut self, _context: &mut Context) -> GameResult {
        // TODO: update game logic here
        Ok(())
    }

    fn draw(&mut self, _context: &mut Context) -> GameResult {
        // TODO: update draw here
        Ok(())
    }
}

pub fn main() -> GameResult {
    // Create a game context and event loop
    let context_builder = ggez::ContextBuilder::new("rust_sokoban", "sokoban")
        .window_setup(conf::WindowSetup::default().title("Rust Sokoban!"))
        .window_mode(conf::WindowMode::default().dimensions(800.0, 600.0))
        .add_resource_path(path::PathBuf::from("./resources"));

    let (context, event_loop) = &mut context_builder.build()?;
    // Create the game state
    let game = &mut Game {};
    // Run the main event loop
    event::run(context, event_loop, game)
}

```

```
➜  rust-sokoban git:(master) ✗ cargo run
   Compiling rust-sokoban v0.1.0 (/Users/wbprice/Documents/rust-sokoban)
error[E0433]: failed to resolve: use of undeclared type or module `path`
  --> src/main.rs:30:28
   |
30 |         .add_resource_path(path::PathBuf::from("./resources"));
   |                            ^^^^ use of undeclared type or module `path`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: could not compile `rust-sokoban`.

To learn more, run the command again with --verbose.
```

Probably just needs a `use path;` at the top.

--- a/code/rust-sokoban-01/src/main.rs
+++ b/code/rust-sokoban-01/src/main.rs
@@ -1,5 +1,6 @@
 use ggez;
 use ggez::{conf, event, Context, GameResult};
+use std::path;
 
 // This struct will hold all our game state
 // For now there is nothing to be held, but we'll add

